pyproject.toml

```diff
@@ -20,4 +20,8 @@ exclude = '''
   | build
   | dist
 )/
-'''
\ No newline at end of file
+'''
+
+[tool.ruff]
+ignore-init-module-imports = true
+typing-modules = ["tidy3d.components.types"] # without this Literal["something fails"]
```

Comment:
Maybe add `line-length = 100` here as well, so that ruff LSP works seamlessly with this configuration.

Reply:
good catch, added